imap/http_dav.c

```diff
@@ -2724,8 +2727,14 @@ int propfind_curprivset(const xmlChar *name, xmlNsPtr ns,
                     flags = PRIV_INBOX;
                 else if (!strcmp(fctx->req_tgt->collection, SCHED_OUTBOX))
                     flags = PRIV_OUTBOX;
+
+                flags |= PRIV_NOSUBCOL;
             }
         }
+        else if (fctx->req_tgt->collection &&
+                 fctx->req_tgt->namespace->id == URL_NS_ADDRESSBOOK) {
+            flags |= PRIV_NOSUBCOL;
+        }
```

Comment:
It's not touched as part of this PR so I can't comment on it directly, but just below this block is:

```
flags += PRIV_CONTAINED;
```

Note that it's `+=` rather than `|=`... I think that could have bad effects if the PRIV_CONTAINED bit was already set?  Looks like it can't be already set at the moment, because `flags` was initialised to 0 at the top of the function and this is the first time PRIV_CONTAINED is mentioned.  But if we ever need to do something with PRIV_CONTAINED earlier, this line could bite us

Reply:
Good catch.  Fixed.